code/modules/client/preferences.dm

```diff
@@ -113,9 +113,18 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 		if(load_preferences() && load_character())
 			return
 
-	key_bindings = deepCopyList(GLOB.keybinding_list_by_key)
-
+	// We don't have a savefile or we failed to load them
 	random_character()
+	addtimer(CALLBACK(src, .proc/load_default_keybindings, C), 3 SECONDS)
+	
+
+/datum/preferences/proc/load_default_keybindings(client/C = usr)
```

Comment:
```suggestion
/datum/preferences/proc/load_default_keybindings(client/C = usr)
```
This won't do anything since you're always calling it with a C which will always be a client.